api/ManifestApp/urls.py

```diff
@@ -10,5 +10,5 @@
     path('',views.index, name="homepage"),
     path('api/', include(router.urls)),
     path('api-auth/', include('rest_framework.urls', namespace='rest_framework')),
-    path('api/all-nodes/', views.NodeList.as_view(), name="all-nodes"),
+    path('api/nodes/', views.NodeList.as_view(), name="nodes"),
```

Comment:
I am confused.. i expected to see

```
path('api/nodes/', views.NodeList.as_view(), name="nodes"),
path('api/nodes/<str:vsn>', views.NodeList.as_view(), name="nodes"),
```

where the first one would return all the nodes and the 2nd one would return only the 1 node specified by `<str:vsn>`

Can you please show an example of the URLs that you would use to 
a) get manifest for all the nodes
b) get the manifest for a single node (by VSN)

Reply:
a) api/nodes/
b) api/nodes/?VSN=W023 -> is this acceptable?

Reply:
I would really like it to be

`api/nodes/`
`api/nodes/<VSN>`  (ex. `api/nodes/W023`)